pkgs/tools/security/wordlists/dirbuster.nix

```diff
@@ -0,0 +1,29 @@
+{ fetchzip
+, lib
+}:
+
+let
+  pname = "DirBuster";
+  version = "1.0-RC1";
+
+  shared = "dirbuster";
+
+in fetchzip rec {
+  name = "${pname}-${version}";
+
+  url = "mirror://sourceforge/dirbuster/DirBuster%20%28jar%20%2B%20lists%29/${version}/${pname}-${version}.zip";
+  sha256 = "sha256-nByEKfhJ5TsZjvL4dTcgrSw0kUUPPVQNJOywdlZwiTo=";
+
+  postFetch = ''
+    mkdir -p $out/share/dirbuster
+    unzip -j $downloadedFile ${pname}-${version}/directory-list-\*.txt -d $out/share/${shared}
+  '';
+
+  passthru = { inherit shared; };
+
+  meta = with lib; {
+    homepage = "https://sourceforge.net/projects/dirbuster/";
+    license = licenses.gpl2;
```

Comment:
gpl2Only or gpl2Plus?

Reply:
I am not too sure how to interpret that at https://sourceforge.net/projects/dirbuster/ ...
I would say `gpl2Only`?

Reply:
I noticed the software is GPL, but the wordlists included with dirbuster themselves seem to be a different license. They all have copyright headers with "Creative Commons Attribution-Share Alike 3.0"

Reply:
The software itself it lgpl2only. If the lists have that header please mark them as that.

Reply:
Oh, good catch @buckley310 !